components/ide/jetbrains/gateway-plugin/src/main/kotlin/io/gitpod/jetbrains/gateway/GitpodWorkspacesView.kt

```diff
@@ -293,7 +300,10 @@ class GitpodWorkspacesView(
                                     )
                                 }
                                 label(getRelativeTimeSpan(info.latestInstance.creationTime))
-                                button("Connect") {
+                                val jbClientConnected = GitpodConnectionProvider.jetbrainsClientMap[info.workspace.id]
```

Comment:
Should not we rather focus existing client instead?

Reply:
This is a good idea, however I haven't found a way to requestFocus for the ThinClient yet